### ISSUE TITLE
Backbone api url is incorrect

### ISSUE THREAD

**Issue by andrewahead4** [WP-API/WP-API#1533]
I had a fresh install of WP and the API to test out the backbone helper script.

Making a call to the backbone script in wp-api results in a broken url for collections and I presume models also.

```
posts = new wp.api.collections.Posts();

console.log( posts.url );
```
results in seeing *".../wp-json//posts"* i.e. the *wp/v2* is missing.

I have tracked this down to extras.php line 34 where the wp_localize_script sets up the WP_API_Settings and hacked it for my personal development

**Comment by rmccue** [comment]
I think you're right, and I just ran into this. The full URL for the posts collection should be `root + 'wp/v2/posts'`.

@tlovett1 @kadamwhite Seem right to you?

**Comment by tlovett1** [comment]
Yea, that seems right.

**Comment by justintadlock** [comment]
Yep, `root + 'wp/v2/posts'` is correct. I just hit this issue too.

**Comment by kadamwhite** [comment]
:+1:

**Comment by rmccue** [comment]
Fixed in #1709, thanks!
